react-app/src/features/package/index.tsx

```diff
@@ -43,9 +43,13 @@ export const DetailsContent = ({ id }: DetailsContentProps) => {
 
   return (
     <div className="w-full py-1 px-4 lg:px-8">
-      <section id="package_overview" className="block md:flex space-x-0 md:space-x-8">
-        <PackageStatusCard submission={submission} />
-        <PackageActionsCard id={id} submission={submission} />
+      <section id="package_overview" className="flex flex-col sm:flex-row sm:justify-between">
+        <div className="w-full -mb-3 sm:mb-0 sm:w-[45%]">
+          <PackageStatusCard submission={submission} />
+        </div>
+        <div className="w-full -mt-3 text-wrap sm:mt-0 sm:text-nowrap sm:min-w-min sm:w-[45%]">
+          <PackageActionsCard id={id} submission={submission} />
+        </div>
```

Comment:
Should we move these elements into their respective components...?

Reply:
I think we can, but that would make them not reusable somewhere else

Reply:
Yeah, I checked before commenting since they're only used once elsewhere

Reply:
but the `CardWithTopBorder` has a margin that I can't change from within the component and it messes up the spacing
![CleanShot 2025-03-25 at 12 57 56](https://github.com/user-attachments/assets/011da639-8280-43c9-afaa-58980dfd5a75)

Reply:
I meant, move the new parent elements you added into `PackageStatusCard` and `PackageActionsCard` respectively. Don't change the stuff that's generalized.

Reply:
the problem is once you are inside the `PackageStatusCard` and `PackageActionCard` you have already been wrapped by `DetailCardWrapper` which I can change because it's only used by those two, but sandwiched in-between those is `CardWithTopBorder` and it has it's own spacing changes

Reply:
I have worked on combating the sizing issues for about an hour and I understand your issues, but this is the best way I can find to handle the spacing stuff.